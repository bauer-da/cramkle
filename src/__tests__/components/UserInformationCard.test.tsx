import * as React from 'react'
import { render } from 'react-testing-library'
import { createStore } from 'redux'

import rootReducer from '../../rootReducer'
import UserInformationCard from '../../components/UserInformationCard'

describe('<UserInformationCard />', () => {
  it('should match snapshot', () => {
    const dummyStore = createStore(rootReducer)

    const component = render(<UserInformationCard store={dummyStore} />)

    expect(component.container).toMatchSnapshot()
  })
})
