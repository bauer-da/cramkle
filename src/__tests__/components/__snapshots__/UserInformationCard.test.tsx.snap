// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserInformationCard /> should match snapshot 1`] = `
<div>
  <div
    class="user-popup"
  >
    <i
      class="material-icons"
    >
      account_circle
    </i>
  </div>
</div>
`;
